Go/Go.sublime-syntax

```diff
@@ -432,28 +432,69 @@ contexts:
     # Dot without integer
     - match: \.\d+(?:{{exponent}}\d+)?
       scope: invalid.deprecated.go
+    # Hex float literal, no fraction
+    - match: (0[xX])(?:_)?\h+(?:_\h+)*([pP])([\+\-])?\d+(?:_\d+)*(i)?
```

Comment:
`(?:_)?` is equivalent to `_?`.

Reply:
Good call. Removed all the unnecessary passive groups.